__tests__/cmds/versions.test.js

```diff
@@ -180,29 +172,24 @@ describe('rdme versions*', () => {
         .basicAuth({ user: key })
         .reply(200, { version });
 
-      return deleteVersion.run({ key, version }).catch(err => {
```

Comment:
This is the test that wasn't actually testing anything because the command was returning a success message instead of throwing an exception.

Reply:
yikes, nice find!